README.md

```diff
@@ -20,7 +20,7 @@ this README. Please refer to the Kubernetes and Helm documentation.
 
 The versions required are:
 
-  * **Helm 3.0+** - This is the earliest version of Helm tested. It is possible
+  * **Helm 3.6+** - This is the earliest version of Helm tested. It is possible
     it works with earlier versions but this chart is untested for those versions.
```

Comment:
```suggestion
  * **Helm 3.6+**
```

I guess this is now more of a hard requirement?

Reply:
Good point!